### DISCUSSION TITLE
Greater Customization for File Output Format

### DISCUSSION THREAD

**Discussion by TrazhCant** [zotify-dev/zotify#37] [Ideas]
I opened an issue regarding this, but maybe that's not appropriate?  Anyway, it would be better to mandate the track/disc #'s be 2-digit values.  That will ensure the file list order is correct when viewing a dir listing.

On that note, can someone explain lines 58-77 of track.py?  I'd gladly just make the modification on my own if I could reference the syntax used there.  It looks like a lot of it is calling multiple lists, but I can't find any python documentation with examples of multiple consecutive single-item lists like that.  I'm interested in Python development, but at the moment I'm just flying by the seat of my pants trying to piecemeal learn bits and pieces like that lol

**Comment by ryansereno** [comment]
```py
album_name = info[TRACKS][0][ALBUM][NAME]
name = info[TRACKS][0][NAME]
release_year = info[TRACKS][0][ALBUM][RELEASE_DATE].split('-')[0]
...
```
        
 this type of syntax is similar to what you would see in javascript;
 they aren't accessing separate lists, but are accessing nested values within the json, returned by the spotify api
 for a better explanation, check this out:
  https://stackoverflow.com/questions/23306653/python-accessing-nested-json-data

 
 check out the spotify api documentation to better understand the structure of the data that is returned:
 https://developer.spotify.com/documentation/web-api/reference/get-playlist
